feeds.conf

```diff
@@ -1,6 +1,7 @@
 src-git packages https://github.com/openwrt/packages.git^56331e808f499610c7a3ee295485ad49a9d351e7
 src-git luci https://github.com/openwrt/luci.git^c1e6a3dd3ae1be749c8608ffec8d9c90362e17db
 src-git routing https://github.com/openwrt-routing/packages.git^b82d7ccd0abe042359c41229090c2ce5d4276950
+src-git wlanslovenija https://github.com/wlanslovenija/firmware-packages-opkg.git^c3ae82b7f1d411ef0cd7ce594697b4167bb00462
 src-git packages_berlin https://github.com/freifunk-berlin/firmware-packages.git^e23be3e9922896c540e8f7d25a36917ce72aff9b
```

Comment:
why adding this feed?
This also defines a tunneldigger-package, we already have via the gluon-feed?

Reply:
Doesn't it make sense to take the package from the main repository and origional developers and not from a forked branch?  The two are different.  Not very different, but still somehow different.  

Also, the tunneldigger-broker which I am working with is from wlanslovenija.  They wrote all the documentation and are in my opinion more responsible for the tunneldigger project.

Reply:
After looking closer, the gluon version sources a more recent version of the client code than the  wlanslovenija version.  And the gluon init script reads more options from the uci config file.

But the gluon version does not seem to be an official fork of the wlanslovenija version.  But more like they just copied the code from them into their repository.  

I don't know any more which one makes more sense.  The official out-of-date package which we could contribute to or the non-official package which does not contribute to the official one.

Reply:
> Also, the tunneldigger-broker which I am working with is from wlanslovenija. They wrote all the documentation and are in my opinion more responsible for the tunneldigger project.

https://github.com/wlanslovenija/firmware-packages-opkg HEAD (https://github.com/wlanslovenija/firmware-packages-opkg/commit/c3ae82b7f1d411ef0cd7ce594697b4167bb00462) is from 26 Jun 2017 ; https://github.com/wlanslovenija/firmware-packages-opkg/blob/c3ae82b7f1d411ef0cd7ce594697b4167bb00462/net/tunneldigger/Makefile#L6 uses tunneldigger with 297592db89c7cb302db7a23e7a4dd5ceef957354 (https://github.com/wlanslovenija/tunneldigger/commit/297592db89c7cb302db7a23e7a4dd5ceef957354) of 11 Oct 2015.
Gluon-packages is using tunneldigger https://github.com/wlanslovenija/tunneldigger/commit/d7db350011076d6a83855d412885a29a7d142b6e of 13 Dec 2017 (https://github.com/freifunk-gluon/packages/blob/6c38a5dd639ee4649bc98401c5b3bb27ff99de4d/net/tunneldigger/Makefile#L9) - just a "bit" more recent.

Reply:
I've updated the branch to include the gluon feed instead of wlanslovenija.